hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/OFSPath.java

```diff
@@ -102,7 +108,12 @@ private void initOFSPath(URI uri, boolean endsWithSlash) {
         // TODO: Make this configurable in the future.
         volumeName = OFS_MOUNT_TMP_VOLUMENAME;
         try {
-          bucketName = getTempMountBucketNameOfCurrentUser();
+          if (conf.getBoolean(OZONE_OM_ENABLE_OFS_SHARED_TMP_DIR,
+              OZONE_OM_ENABLE_OFS_SHARED_TMP_DIR_DEFAULT)) {
+            bucketName = OFS_MOUNT_NAME_TMP;
```

Comment:
Do we want to make the shared temp bucket name configurable?

Right now the value of `OFS_MOUNT_NAME_TMP` happens to be `tmp`, but the meaning of this constant is different.

Reply:
> Do we want to make the shared temp bucket name configurable?

Filing a jira, HDDS-7746 -
https://issues.apache.org/jira/browse/HDDS-7746

> Right now the value of OFS_MOUNT_NAME_TMP happens to be tmp, but the meaning of this constant is different. Perhaps define a new constant?

Thanks.  Defined `OFS_SHARED_TMP_BUCKETNAME` as constant used for setting tmp dir bucketname to "tmp".